### PR TITLE
remove quickstart ref

### PR DESCRIPTION
**Pull request by snecklifter**

### CODE REVIEW

**Comment by snecklifter** [comment]
The agreement was to remove the quickstart ref from all Packstack stuff so amended index to match.

File: source/install/packstack.html.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Packstack quickstart
+title: Packstack
 category: documentation
 authors: apevec, dneary, garrett, jasonbrooks, jlibosva, mmagr, pixelbeat, pmyers,
   rbowen, gbraad, cbrown2
 wiki_category: Documentation
-wiki_title: Quickstart
+wiki_title: Packstack
```
Comment (apevec) [inline source/install/packstack.html.md]: This is historical info from wiki to website migration and it is not used anywhere afaik. @rbowen maybe we could cleanup all those wiki_* fields now?

Comment (duck-rh) [inline source/install/packstack.html.md]: I think it was to keep trace of the origin, but it's old now and nobody cares, so I'm also in favor of removing them. Also writers keep misunderstanding and adding wiki headers for new content so…

Comment (snecklifter) [inline source/install/packstack.html.md]: @apevec happy to abandon this one and just do one patch that removes these fields?

Comment (rbowen) [inline source/install/packstack.html.md]: Yes, agreed. We should just drop all of the wiki_title fields. They no longer serve any purpose.

Comment (duck-rh) [inline source/install/packstack.html.md]: I checked the Ruby magic and `wiki_title` is used in `config.rb`. Fortunately this is not needed anymore and will save us some build time :-)

So could you finish this PR, removing this part of the change, and I'm going to create another one for the cleanup ?

**Comment by duck-rh** [comment]
Seems fine but you need to rebase and fix conflicts.

@apevec could you recheck your review please?

**Comment by rbowen** [comment]
I'm not sure what to do with this. We want the changes, but it keeps getting further out of sync, and I have had no luck trying to resolve the conflicts. Can someone with more Git-fu help out here?

**Comment by snecklifter** [comment]
@rbowen I'm pretty sure this is just waiting on @apevec to clear his block / red x

**Pull request closed without merging**